utils/pretty_printers.cc

```diff
@@ -11,16 +11,20 @@
 namespace utils {
 
 std::ostream& operator<<(std::ostream& os, pretty_printed_data_size data) {
-    static constexpr const char *suffixes[] = {" bytes", "kB", "MB", "GB", "TB", "PB"};
-
-    unsigned exp = 0;
-    while ((data._size >= 1000) && (exp < sizeof(suffixes))) {
-        exp++;
-        data._size /= 1000;
+    static constexpr const char *const suffixes[] = {" bytes", "kB", "MB", "GB", "TB", "PB"};
```

Comment:
Only the first "const" was needed, the const at the end is not needed (it means the pointer itself is const), but doesn't hurt either, so ok...
In any case, there was nothing wrong with the original line here before your patch :-)

I'll let the CI check this before committing.

Reply:
okay. thought it was more correct. since i was staring it for too long..

Reply:
Yes, it's correct, in the sense that you can also write "const int a = f();" if the code below doesn't change the value of a. But you don't have it write it, and usually you don't. The other const is the more important one, because the compiler can't affort to let you write into the **content** of a string constant, otherwise it can cause funny bugs (I had such a bug in the past in a Fortran program, long story ;-)).